getting-started/kubernetes/helm.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Installing Calico with Helm on Kubernetes
+description: Install Calico on a Kubernetes cluster using Helm 3.
+canonical_url: '/getting-started/kubernetes/helm'
+---
+
+### Big picture
+
+This installs Calico on a Kubernetes cluster using Helm 3.
+
+### Value
+
+Helm charts are a way to package up an application for Kubernetes.  This can be thought of as similar to the way `apt` or `yum` packages applications for operating systems.  
```

Comment:
Helm charts are a way to package up an application for Kubernetes (similar to the way `apt` or `yum` packages applications for operating systems).  Helm is also used by tools like ArgoCD to manage applications in a cluster, taking care of install, upgrade (and rollback if needed), etc.

Reply:
```suggestion
Helm charts are a way to package up an application for Kubernetes (similar to `apt` or `yum` for operating systems). Helm is also used by tools like ArgoCD to manage applications in a cluster, taking care of install, upgrade (and rollback if needed), etc.
```